### PR TITLE
Removes the spirit realm runes ability to summon cult ghosts.

### LINKED ISSUE #15392: Cult "Spirit Realm" rune doesnt ask for consent
**Issue Description**:
The cult "Spirit Realm" rune does not ask ghosts if they want to actually be spawned in. 

**What did you expect to happen**: 
There to be a preference for this or some popup or something else

**What happened instead**:
I got randomly spawned in with no warning or no question, just because I wanted to orbit a cultist

**Why is this bad/What are the consequences**:
Some people just want to ghost, and not be spawned in randomly with no warning. Ghosts should be given a choice of when they are spawned in.

**Steps to reproduce the problem**:
Orbit a cultist whos using a "spirit realm" rune

**When did the problem start happening**:
Likely since cult refactor

**Extra information**:
N/A

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
title.
also changes the type path of the rune from `manifest` to `spirit` as it no longer manifests anything.
Closes #15392 

## Why It's Good For The Game
Cult is incredibly overloaded, with tons of tools to accomplish their objectives. this results in cultsits not needing to work together to achieve them.

Cult is supposed to be a team antagonist but the ability to create a team on the fly subverts that idea. theres no need to work together.

The removal forces cult to actually act as a team.

It also has the added benefit of preventing the cheese strat where you summon 4 ghosts, send them at security and keep spawning them until they die.

## Changelog
:cl:
del: Cultists can no longer summon ghost cultists using the spirit realm rune.
/:cl:

### CODE REVIEW

**Comment by matttheficus** [comment]
My only note to this is how Cult will now fall even faster unless you get an experienced round start team. I’d honestly prefer it loses it’s free aghost and make the person using the rune listen to dchat to find targets/get info/etc.

That and (don’t know if it’s possible, wheeeeee antag code) make it so cult ghosts can’t sacc mindshield roles. Would make it so you need proper bodies to do the more “difficult” sacc.

**Comment by Qwertytoforty** [comment]
posting from discord:
While wanting to increase cult teamwork is good, I am not sure this is the way to do it. Runes have notoriously been hard to get new cultists to stand near to get going. Making ghosts go away isn't going to improve that. While I get the concern for speed robes fast converting interaction, I don't think this will stop it, instead it will lead to officers or some unlucky bald player getting sharded and used as a ghost. As for the 4 ghost sec raid stratigy, I have not seen it for a few months, and while perhapd this should be log dived for to confirm, I don't think it will be an issue anymore. 
Finaly, the ghosts are often needed for cult to delay / distract the flashbang scattershot durand breaking down the cult base, that cult otherwise really can not do something about.
In conclusion, I would suggest limiting it to 2 ghosts, and lowering that by 1 if the first sacrifice is done, and by 1 if their is 8-12 cultists. That way, if the cult is low on numbers they will still be able to make up for it if sec is going hard, and killing off cult fast, to keep up the conversions to a certain point....

...making cult ghosts go away is not going to make people work as a team better. It will simply lead to people trying to get around it, telling new cultists to hide in a locker till I come back with someone, or will lead to nothing at all, with cult acting the same, losing a lot of manpower otherwise

**Comment by Carthusia** [comment]
I like the idea that if the cult is low on manpower they can summon more ghosts, makes it so that if even one cultist remains, they could restart the cult, or even be a decent threat on their own.

**Comment by RagingLemons1** [comment]
if you're trying to nerf cult ghost sec raids just buff the amount of damage/tick that summoning spirit ghosts costs. removing them outright is kinda cringe I think. Like that other dude said, this rune lets the cult restart from one person.

**Comment by hal9000PR** [comment]
Clearly people aren't fond of this change.
I will do a revised version of this after we get some data from #17893
